app/utils/icons.js

```diff
@@ -22,9 +22,9 @@ const store = {
   'plus': `<svg style="width:100%;height:100%" viewBox="0 0 24 24">
     <path fill="currentColor" d="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z" />
   </svg>`, // materialdesignicons.com plus
-  'check': `<svg style="width:24px;height:24px" viewBox="0 0 24 24">
-    <path fill="currentColor" d="M21,7L9,19L3.5,13.5L4.91,12.09L9,16.17L19.59,5.59L21,7Z" />
-  </svg>`,
+  // 'check': `<svg style="width:24px;height:24px" viewBox="0 0 24 24">
```

Comment:
I wasn't sure if maybe you had a reason for this so I just commented out the duplicate `check` property rather than deleting it